.github/workflows/backend.yml

```diff
@@ -4,10 +4,10 @@ on: [workflow_dispatch, push, pull_request]
 
 jobs:
   run:
-    uses: flarum/framework/.github/workflows/REUSABLE_backend.yml@1.x
+    uses: flarum/framework/.github/workflows/REUSABLE_backend.yml@2.x
     with:
       enable_backend_testing: true
       enable_phpstan: true
-      php_versions: '["8.0", "8.1", "8.2", "8.3"]'
+      php_versions: '["8.2", "8.3", "8.4"]'
```

Comment:
For `2.x` we don't need to specify the PHP versions here, so this line can be removed